docs/runbook.md

```diff
@@ -0,0 +1,266 @@
+or its equivalent.
+
+## Snapshots
+
+Snapshots can be fetched by URL but as of early 2022 are approaching 300MiB.
+
+Fetch its metadata:
+
+```bash
+wget -N https://snapshots.helium.wtf/mainnet/latest-snap.json
+```
+
+The resulting JSON contains `height`, which gets used for file name of
+snapshot to download.
+
+```json
+{
+    "compressed_hash": "SGflG0oH0TTbairR3JCbtXoGJMhl6pYW4dHExmydRIc",
+    "compressed_size": 262995691,
+    "file_hash": "k0Pp00Y2wIeDagvzFMz66RAbD73e2y8fisCibP2Au2M",
+    "file_size": 427285643,
+    "hash": "3MmAnIgap0AMXL8n9eixxvnN8U8Qwtc3X_N6XXY1oR0",
+    "height": 1279441
+}
+```
+
+Craft the new URL based upon that height:
+
+```bash
+wget -N https://snapshots.helium.wtf/mainnet/snap-1279441.gz
+```
+
+However, those are **generated** from the chain approximately **every 11 hours**.
+
+Therefore, compare chain heights or file timestamps.
+
+```bash
+curl https://api.helium.io/v1/blocks/height
+```
+
+Or:
+
+```bash
+curl --head https://snapshots.helium.wtf/mainnet/snap-1279441.gz
+TZ=Z date
+```
+
+Scripted version:
+
+```bash
+wget -N \
+  $(curl https://snapshots.helium.wtf/mainnet/latest-snap.json | \
+    jq .height | \
+    sed 's%^\(.*\)$%https://snapshots.helium.wtf/mainnet/snap-\1.gz%)'
+```
+
+The base of that URL is specified by `blockchain` `->` `snap_source_base_url`
+within
+[`sys.config.src`](../config/sys.config.src)
+with the filename being then-current block height.
+
+Similarly, locally generated snapshots should be within the directory path
+specified by `blockchain` `->` `base_dir`.
+
+### Manual Snapshots
+
+Start on a server confirmed to be current with respect to blockchain height.
+
+Maybe start on a validator, which uses the `minor` command rather than
+`router` but otherwise works the same.
+
+```bash
+miner snapshot take latest.snap
+docker cp validator:/opt/miner/latest.snap /tmp/
+rsync -a /tmp/latest.snap  192.168.123.123:/var/data/latest.snap
+```
+
+Once copied to the intended server, load that snapshot:
+
+```bash
+router snapshot load /var/data/latest.snap
+```
+
+## State Channels
+
+State Channels provide off-chain transactions to later be durably recorded
+on-chain en masse and are critical for timely operations and network
+semantics.
+
+For a router that is actively used, there will always be at least one State
+Channel active.
+
+Get summary of State Channels currently in use on router or
+packet-purchaser, respectively.
+
+```bash
+router sc list
+pp sc list
+```
+
+Or:
+
+```bash
+tail -F /var/data/log/router.log | grep _sc_worker
+tail -F /var/data/log/packet_purchaser.log | grep _sc_worker
+```
+
+## Tracing A Device
+
+Traces disable themselves after 240 minutes or if the service gets
+restarted.
+
+```bash
+router device trace --id=abc12345-bbbb-cccc-dddd-eeeeeeeeeeee
+tail -F /var/data/router/log/traces/abc12.log
+```
+
+## Erlang Expressions
+
+For one-off expressions **not** intended to impact the running BEAM, use `eval`:
```

Comment:
I know of the chat history that influences the language choice here, but the context will be lost shortly:  suggest changing or augmenting "impact" to something like "persist inside" for clarity

Reply:
Yes, that's better wording.  Fixed in b4aba82